alicloud/table_alicloud_ecs_eni.go

```diff
@@ -0,0 +1,290 @@
+			},
+			{
+				Name:        "zone_id",
+				Description: "The zone ID of the ENI.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "private_ip_address",
+				Description: "The private IP address of the ENI.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "mac_address",
+				Description: "The MAC address of the ENI.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "instance_id",
+				Description: "The ID of the instance to which the ENI is bound.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "service_id",
+				Description: "The ID of the distributor to which the ENI belongs.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ServiceID"),
+			},
+			{
+				Name:        "service_managed",
+				Description: "Indicates whether the user is an Alibaba Cloud service or a distributor.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "resource_group_id",
+				Description: "The ID of the resource group to which the ENI belongs.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "owner_id",
+				Description: "The ID of the account that owns the ENI.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "security_groupIds",
+				Description: "The IDs of the security groups to which the ENI belongs.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "associated_public_ip",
+				Description: "The public IP address associated with the secondary private IP address of the ENI.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "attachment",
+				Description: "Attachments of the ENI",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "private_ip_sets",
+				Description: "The private IP addresses of the ENI.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "ipv6_sets",
+				Description: "The IPv6 addresses assigned to the ENI.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Ipv6Sets"),
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags attached with the resource.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags.Tag").Transform(modifyEcsSourceTags),
+			},
+
+			// steampipe standard columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags.Tag").Transform(ecsTagsToMap),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEcsEniAka,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("NetworkInterfaceName"),
```

Comment:
Add fallback mechanism to store `NetworkInterfaceId` as `title`, if `NetworkInterfaceName` is not provided.